README.md

```diff
@@ -1,10 +1,36 @@
 # ManagementPortal
 
+[![Build Status](https://travis-ci.org/RADAR-CNS/ManagementPortal.svg?branch=dev)](https://travis-ci.org/RADAR-CNS/ManagementPortal)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/87bb961266d3443988b52ee7aa32f100)](https://www.codacy.com/app/RADAR-CNS/ManagementPortal?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=RADAR-CNS/ManagementPortal&amp;utm_campaign=Badge_Grade)
+
 ManagementPortal is an application which is used to manage pilot studies for [RADAR-CNS](http://www.radar-cns.org/).
 
+## Quickstart
+
+The quickest way to get ManagementPortal up and running in production mode is by using the included
+docker-compose files. 
+1. First, we need to generate a key pair for signing JWT tokens as follows:
+```shell
+keytool -genkey -alias selfsigned -keyalg RSA -keystore src/main/resources/config/keystore.jks -keysize 4048 -storepass radarbase
+```
+**Make sure the key password and store password are the same!** This is a requirement for Spring Security.
+
+2. Then, make sure [Docker][], [Docker-Compose][], [Node.js][] and [Yarn][] are installed on your system.
```

Comment:
If the release will have a corresponding image on docker hub, please mention it here